### PR TITLE
Add elements to Login View.

### PR DESCRIPTION
**Pull request by kate-novik**
Fill log in form with necessary elements.
Add SVG social network icons to log in page.
image
image

### CODE REVIEW

**Comment by AlexOreshkevich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486&bid=16027625)

File: frontend/src/views/login/login-view.js [hunk 1]
```diff
@@ -1,13 +1,39 @@
 import {PolymerElement} from "@polymer/polymer/polymer-element.js";
+import '@polymer/iron-iconset-svg/iron-iconset-svg.js';
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Strings must use doublequote.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486)

File: frontend/src/views/login/login-view.js [hunk 2]
```diff
@@ -1,13 +1,39 @@
...
+import '@polymer/iron-icon/iron-icon.js';
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Strings must use doublequote.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486)

**Comment by AlexOreshkevich** [comment]
Please provide screenshots of your changes (while we are resolving issues with deployments for `master` and PR branches).

**Comment by AlexOreshkevich** [comment]
@alabotski please review design of UI for following best practices

**Comment by AlexOreshkevich** [comment]
@kate-novik please consider rebasing from the following guide https://vaadin.com/learn/tutorials/securing-your-app-with-spring-security/form-based

**Comment by AlexOreshkevich** [comment]
https://github.com/vaadin-learning-center/spring-secured-vaadin/tree/polymer-form

**Comment by AlexOreshkevich** [comment]
LoginView should be RWA (Responsive Web Application). Please attach screenshots for small resolutions as well (could be configured on desktop Chrome) for proving that it would work and look well on mobile phones.

**Comment by AlexOreshkevich** [comment]
I would expect to get something like this:

image

Full screen, responsive dialog box that couldn't be avoided, designed like a form to be able to send authentication data to the server, that should look well on different window resolutions (desktop, mobile).

**Comment by AlexOreshkevich** [comment]
Another great example, even preferred one:

image

**Comment by AlexOreshkevich** [comment]
image

File: src/main/java/com/rednavis/vaadin/showcase/views/ConstantUtils.java [hunk 3]
```diff
@@ -6,4 +6,10 @@
 
   public static final String PAGE_LOGIN_TITLE = "Login";
   public static final String PAGE_LOGIN_URL = "loginView";
+  public static final String LOGIN_FORM_LOGIN = "Username";
```
Comment (alabotski) [inline src/main/java/com/rednavis/vaadin/showcase/views/ConstantUtils.java]: This class is for application internal use only.
Localization must be moved to a separate class

File: src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java [hunk 4]
```diff
@@ -34,14 +41,47 @@
   @Id("content")
   private Div content;
 
+  private TextField userName;
```
Comment (alabotski) [inline src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java]: I think we need move field declarations to template
https://vaadin.com/docs/v14/flow/polymer-templates/tutorial-template-intro.html

Something like this
```
    <div class="row">
      <vaadin-text-field id="field" class="field" placeholder="[[fieldPlaceholder]]" value="{{fieldValue}}" on-focus="_onFieldFocus" 
        on-blur="_onFieldBlur" theme="white">
        <iron-icon icon\$="[[fieldIcon]]" slot="prefix"></iron-icon>
      </vaadin-text-field>
      <vaadin-checkbox class="checkbox desktop" checked="{{checkboxChecked}}" on-focus="_onFieldFocus" 
        on-blur="_onFieldBlur">[[checkboxText]]</vaadin-checkbox>
      <vaadin-button id="clear" class="clear-btn" theme="tertiary">
        [[clearText]]
      </vaadin-button>
      <vaadin-button id="action" class="action-btn" theme="primary">
        <iron-icon icon\$="[[buttonIcon]]" slot="prefix"></iron-icon>
        [[buttonText]]
      </vaadin-button>
    </div>
```

You can download good examples here
https://vaadin.com/start/latest

Also we can separate template and styles
https://vaadin.com/docs/v14/flow/importing-dependencies/tutorial-include-css.html
https://vaadin.com/docs/v14/flow/theme/theming-overview.html

File: src/main/java/com/rednavis/vaadin/showcase/views/ConstantUtils.java (hunk 3 above)
Comment (kate-novik) [reply src/main/java/com/rednavis/vaadin/showcase/views/ConstantUtils.java]: Yes, I know, it was added constants without settings of localisation now. And it is another task.

File: frontend/src/views/login/login-view.js [hunk 5]
```diff
@@ -1,18 +1,391 @@
...
+import "@vaadin/vaadin-text-field/vaadin-password-field";
+import "@vaadin/vaadin-ordered-layout/vaadin-vertical-layout";
 import {html} from "@polymer/polymer/lib/utils/html-tag.js";
 
 class LoginView extends PolymerElement {
+
+static get properties() {
+
+  return {
+    selected: {
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Unnecessary block.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486)

**Comment by AlexOreshkevich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486&bid=16027625)

File: frontend/src/views/login/login-view.js [hunk 6]
```diff
@@ -1,18 +1,391 @@
...
+import "@vaadin/vaadin-text-field/vaadin-text-field";
+import "@vaadin/vaadin-text-field/vaadin-password-field";
+import "@vaadin/vaadin-ordered-layout/vaadin-vertical-layout";
 import {html} from "@polymer/polymer/lib/utils/html-tag.js";
 
 class LoginView extends PolymerElement {
+
+static get properties() {
+
+  return {
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Unnecessary block.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486)

File: src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java [hunk 7]
```diff
@@ -4,13 +4,9 @@
...
 import static com.rednavis.vaadin.showcase.views.ConstantUtils.VIEW_PORT;
 
-import com.rednavis.vaadin.showcase.views.dashboard.DashboardView;
-import com.vaadin.flow.component.ClickEvent;
-import com.vaadin.flow.component.ComponentEventListener;
 import com.vaadin.flow.component.Tag;
-import com.vaadin.flow.component.UI;
-import com.vaadin.flow.component.button.Button;
 import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.dependency.NpmPackage;
```
Comment (AlexOreshkevich) [inline src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java]: Codacy Issue found: [Avoid unused imports such as 'com.vaadin.flow.component.dependency.NpmPackage'](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486)

File: frontend/src/views/login/login-view.js [hunk 8]
```diff
@@ -1,18 +1,391 @@
...
+    selected: {
+      type: Number,
+      value: 0,
+      notify: true,
+      reflectToAttribute: true
+    }
+  }
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486)

**Comment by AlexOreshkevich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486&bid=16027625)

File: frontend/src/views/login/login-view.js [hunk 9]
```diff
@@ -1,18 +1,347 @@
...
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486)

File: frontend/src/views/login/login-view.js [hunk 10]
```diff
@@ -1,18 +1,347 @@
...
+import "@vaadin/vaadin-text-field/vaadin-password-field";
+import "@vaadin/vaadin-ordered-layout/vaadin-vertical-layout";
 import {html} from "@polymer/polymer/lib/utils/html-tag.js";
 
 class LoginView extends PolymerElement {
+
+static get properties() {
+
+  return {
+    selected: {
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Unnecessary block.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486)

File: frontend/src/views/login/login-view.js [hunk 11]
```diff
@@ -1,18 +1,347 @@
...
+import "@vaadin/vaadin-text-field/vaadin-text-field";
+import "@vaadin/vaadin-text-field/vaadin-password-field";
+import "@vaadin/vaadin-ordered-layout/vaadin-vertical-layout";
 import {html} from "@polymer/polymer/lib/utils/html-tag.js";
 
 class LoginView extends PolymerElement {
+
+static get properties() {
+
+  return {
```
Comment (AlexOreshkevich) [inline frontend/src/views/login/login-view.js]: Codacy Issue found: [Unnecessary block.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4821486)

File: src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java (hunk 4 above)
Comment (kate-novik) [reply src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java]: Yes, I agree with a point of creating dom elements in js and I re-work PR.

File: src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java [hunk 12]
```diff
@@ -35,30 +24,10 @@
 @Viewport(VIEW_PORT)
 public class LoginView extends PolymerTemplate<TemplateModel> {
```
Comment (AlexOreshkevich) [inline src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java]: Please provide some description and link to the corresponding js template

**Pull request merged**